### PR TITLE
testprojectlibrary: cleaning up reports, setting return values to be …

### PR DESCRIPTION
**Pull request by Rantzur1992**
…more clear.

Signed-off-by: rantz <ran.tzur@testproject.io>

### CODE REVIEW

File: TestProjectLibrary/TestProjectLibrary.py [hunk 1]
```diff
@@ -604,63 +609,76 @@ def table_should_contain(self, locator, expected, loglevel="TRACE"):
...
     @keyword
     def wait_until_element_does_not_contain(self, locator, text, timeout=None, error=None):
-        self.base(
-            locator, f"Element {locator} does not contain {text} under {timeout} seconds", f" {text}", text, timeout, error
-        )
+        message = self._set_message(timeout)
+        self.base(locator, f"Element {locator} does not contain {text} under {message}", f" {text}", text, timeout, error)
+
+    def _set_message(self, timeout):
+        return "set default timeout" if timeout is None else f"{timeout} seconds"
```
Comment (vitalybu) [inline TestProjectLibrary/TestProjectLibrary.py:681]: Are you sure the timeout is always given in seconds?
If timeout is `None` the message will appear as:
`Element {locator} does not contain {text} under set default timeout`?

Comment (Rantzur1992) [reply TestProjectLibrary/TestProjectLibrary.py:681]: Yes, that is the message.
Regarding the seconds, that is what their documentation shows.

Comment (vitalybu) [reply TestProjectLibrary/TestProjectLibrary.py:681]: What documentation?
IMO this message is not very informative, so why not improve it?
If we know what the default timeout is, why not add it to the message?

Comment (Rantzur1992) [reply TestProjectLibrary/TestProjectLibrary.py:681]: https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Set%20Selenium%20Timeout
I will check although I think it is set to 0 by default

Comment (Rantzur1992) [reply TestProjectLibrary/TestProjectLibrary.py:681]: @vitalybu  The default value is set to 0.
This IMO looks like just a better way of presenting it.

Comment (vitalybu) [reply TestProjectLibrary/TestProjectLibrary.py:681]: Do you really think this message is easy to understand?
`Element //... does not contain bla under set default timeout`

Comment (Rantzur1992) [reply TestProjectLibrary/TestProjectLibrary.py:681]: What do you suggest?

Comment (vitalybu) [reply TestProjectLibrary/TestProjectLibrary.py:681]: If no timeout is defined, not need to print anything:
`Element {locator} does not contain {text}`

If a timeout was set:
`Element {locator} does not contain {text} (timeout: {timeout} seconds)`

Comment (Rantzur1992) [reply TestProjectLibrary/TestProjectLibrary.py]: Done

**Review by vitalybu — APPROVED** [review]

**Pull request merged**